src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  const clickHandler = id => {
```

Comment:
```suggestion
  const handleClick = id => {
```
In react developers have a convention how to name handlers
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364